<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Livros_em_baixa</class>
 <widget class="QDialog" name="Livros_em_baixa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LIVROS EM BAIXA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>VISUALIZAR DADOS </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ORDENAR LIVROS POR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Indice</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nome_livro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nome_autor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trilha</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CDU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Editora</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Colecao</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data_edic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n_exemplar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>situacao</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>data_entrada</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>data_emprestimo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>data_devolucao</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NÃšMERO DE LIVROS EM BAIXA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
